src/test/java/com/alipay/oceanbase/rpc/ObTableClientAutoIncTest.java

```diff
@@ -0,0 +1,497 @@
+
+            tableQuery.select("c1", "c2", "c3");
+            filter = new ObTableValueFilter(ObCompareOp.EQ, "c1", 6);
+            tableQuery.setFilter(filter);
+            result = tableQuery.execute();
+            Assert.assertTrue(result.next());
+            value = result.getRow();
+            Assert.assertEquals(6, value.get("c2"));
+            Assert.assertEquals(104, value.get("c3"));
+
+            // test increment exist, increment
+            value = client.increment("test_auto_increment_not_rowkey", new Object[] {6},
+                    new String[] { "c2" }, new Object[] { 6 }, true);
+
+            tableQuery.select("c1", "c2", "c3");
+            filter = new ObTableValueFilter(ObCompareOp.EQ, "c1", 6);
+            tableQuery.setFilter(filter);
+            result = tableQuery.execute();
+            Assert.assertTrue(result.next());
+            value = result.getRow();
+            Assert.assertEquals(12, value.get("c2"));
+            Assert.assertEquals(104, value.get("c3"));
+
+            // test illegal increment on auto increment column
+            try {
+                value = client.increment("test_auto_increment_not_rowkey", new Object[] {6},
+                        new String[] { "c3" }, new Object[] { 1 }, true);
+            } catch (ObTableException e) {
+                assertNotNull(e);
+                assertEquals(ResultCodes.OB_NOT_SUPPORTED.errorCode, e.getErrorCode());
+            }
+
+            // test append not exist, insert
+            Map<String, Object> res = client.append("test_auto_increment_not_rowkey", new Object[] { 200 },
+                    new String[] { "c4" },  new Object[] { "a" } , true);
+
+            tableQuery = client.query("test_auto_increment_not_rowkey");
+            tableQuery.select("c1", "c2", "c3", "c4");
+            filter = new ObTableValueFilter(ObCompareOp.EQ, "c1", 200);
+            tableQuery.setFilter(filter);
+            result = tableQuery.execute();
+            Assert.assertTrue(result.next());
+            value = result.getRow();
+            Assert.assertEquals(106, value.get("c3"));
+            Assert.assertEquals("a", value.get("c4"));
+
+            // test append exist, append
+            res = client.append("test_auto_increment_not_rowkey", new Object[] { 200 },
+                    new String[] { "c4" },  new Object[] { "b" } , true);
+
+            tableQuery = client.query("test_auto_increment_not_rowkey");
+            tableQuery.select("c1", "c2", "c3", "c4");
+            filter = new ObTableValueFilter(ObCompareOp.EQ, "c1", 200);
+            tableQuery.setFilter(filter);
+            result = tableQuery.execute();
+            Assert.assertTrue(result.next());
+            value = result.getRow();
+            Assert.assertEquals(106, value.get("c3"));
+            Assert.assertEquals("ab", value.get("c4"));
+        } finally { // use sql to drop table
+            statement.execute("drop table test_auto_increment_not_rowkey");
```

Comment:
Make it a function, like private `void dropTable(tableName) {}`

Reply:
fixed